tests/support/server/server.coffee

```diff
@@ -4,7 +4,13 @@ wss = new WebSocketServer(port: CONFIG.REALLY_PORT)
 
 wss.on 'connection', (ws) ->
   ws.on 'message', (message) ->
-    ws.send message
-    console.log "Received #{message}"
+    msg = JSON.parse message
+    if msg.cmd is 'error'
```

Comment:
it's better to use another key instead of `cmd` because `cmd` is already used in protocol